supportedBackends/google/pipelines/v1alpha2/src/test/scala/cromwell/backend/google/pipelines/v1alpha2/api/StatusResponseParsingSpec.scala

```diff
@@ -33,15 +33,15 @@ class StatusResponseParsingSpec extends FlatSpec with Matchers with MockitoTrait
     val op = new Operation()
 
     val exception = intercept[RuntimeException](GetRequestHandler.interpretOperationStatus(op))
-    exception.getMessage should be("Caught NPE while processing operation null: {}")
+    exception.getMessage should startWith("Caught NPE while processing operation null")
   }
 
   it should "catch and wrap null pointer exceptions in a name only operation" in {
     val op = new Operation()
     op.setName("my/customName")
 
     val exception = intercept[RuntimeException](GetRequestHandler.interpretOperationStatus(op))
-    exception.getMessage should be("Caught NPE while processing operation my/customName: {name=my/customName}")
+    exception.getMessage should startWith ("Caught NPE while processing operation my/customName")
```

Comment:
This change appears to be removing the core part of the assertion somewhat?

Reply:
A Google library was upgraded which caused the formatting of part of this error message to change. The prefix of the message is still the same so this code matches on that now.